src/test/java/com/jaws/entity/StudyTest.java

```diff
@@ -0,0 +1,43 @@
+package com.jaws.entity;
+
+import com.jaws.entitiy.Member;
+import com.jaws.entitiy.Study;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class StudyTest {
+
+    @Test
+    void studyTest() throws Exception {
+        //given
+        Member member = createMember();
+
+        //when
+        Study study = new Study(member);
+
+        //then
+        assertThat(member.getNickname()).isEqualTo(study.getMember().getNickname());
+        assertThat(member.getStudies().size()).isEqualTo(1);
+        assertThat(member.getStudies().contains(study)).isTrue();
```

Comment:
테스트당 assert문은 하나만 있는 게 좋을 것 같아요!
불가피할 경우 `assertAll()`이나 `assertSoftly()`쓰는 게 좋다고 하네용


💡
[테스트를 작성하는 방법](https://blog.kingbbode.com/52)

Reply:
오 감사합니다!! 아는게 저거밖에 없어서 저렇게 줄줄 쓰면서도 좀 그렇다 생각했는데... ㅋㅋㅋ 테스트도 더 공부해야겠어요

Reply:
`hashCode()`, `equals()` 추가해서 이런식으로 객체끼리 비교하라고 배운 것 같은데.. 우테코랑 많이 다르네요
저도 공부해야겠어요 ㅠㅠ 
```suggestion
        Member updatedMember= new Member("변경된 승호");
        assertThat(member).isEqualTo(updatedMember);
```